### PR TITLE
Typeorm mahta

### PR DESCRIPTION
**Pull request by MahtaMirzaei**
Related: #32

### CODE REVIEW

File: Mahta/TS/TS-typeorm/src/entity/dataSource.ts [hunk 1]
```diff
@@ -0,0 +1,23 @@
+import { Entity, PrimaryGeneratedColumn, Column } from "typeorm";
+
+@Entity()
+export class user {
+  @PrimaryGeneratedColumn()
+  id!: number;
```
Comment (fatemeh-ra) [inline Mahta/TS/TS-typeorm/src/entity/dataSource.ts:6]: why did you use `!`?

File: Mahta/TS/TS-typeorm/src/entity/dataSource.ts [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+  @Column()
+  fName: string;
+
+  @Column()
+  lName: string;
+
+  @Column()
+  age: number;
+
+  constructor() {
```
Comment (fatemeh-ra) [inline Mahta/TS/TS-typeorm/src/entity/dataSource.ts:17]: Does it need a constructor?

File: Mahta/TS/TS-typeorm/src/index.ts [hunk 3]
```diff
@@ -0,0 +1,36 @@
+import { DataSource } from "typeorm";
+import "reflect-metadata";
+import { user } from "./entity/dataSource"; 
+
+const AppDataSource = new DataSource({
+  type: "sqlite",
+  database: "database.db",
+  synchronize: true,
```
Comment (fatemeh-ra) [inline Mahta/TS/TS-typeorm/src/index.ts:8]: Remove the database and test the migrations with `synchronize: false`.

**Review by fatemeh-ra — CHANGES_REQUESTED** [review]

**Comment by mkermani144** [comment]
Closing as stale

**Pull request closed without merging**